wrappers/services.go

```diff
@@ -560,8 +560,7 @@ X-Snappy=yes
 [Timer]
 Unit={{.ServiceFileName}}
 {{ range .Schedules }}OnCalendar={{ . }}
-{{- end }}
```

Comment:
Does this mean we used to generate a broken service file that needs to be updated?

Reply:
The piece of code that did this got merged just today. On top of this, the snap won't install as systemd will reject the timer file we produced.